src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -133,29 +139,23 @@ Template:
+| T2 weighted images                         | T2w              | In arbitrary units (abitrary). The contrast of these images is mainly determined by spatial variations in the (true) transverse relaxation time of the imaged specimen. In spin-echo sequences this contrast is achieved at relatively long repetition and echo times. Generally, gradient echo sequences are not the most suitable option for achieving T2 weighting, as their contrast natively depends on T2-star rather than on T2.        |
+| Proton density (PD) weighted images        | PDw              | In arbitrary units (abitrary). The contrast of these images is mainly determined by spatial variations in the spin density (1H) of the imaged specimen. In spin-echo sequences this contrast is achieved at short repetition and long echo times. In a gradient-echo acquisition, PD weighting dominates the contrast at long repetition and short echo times, and at small flip angles.|
+| T2star weighted images                     | T2starw          | In arbitrary units (abitrary). The contrast of these images is mainly determined by spatial variations in the (observed) transverse relaxation time of the imaged specimen. In spin-echo sequences, this effect is negated as the excitation is followed by an inversion pulse. The contrast of gradient-echo images natively depends on T2-star effects. However, for T2-star variation to dominate the image contrast, gradient-echo acquisitions are carried out at long repetition and echo times, and at small flip angles.                                                                                                     |
+| Fluid attenuated inversion recovery images | FLAIR            | In arbitrary units (abitrary). Structural images with predominant T2 contribution (a.k.a T2-FLAIR), in which signal from fluids (e.g., CSF) is nulled out by adjusting inversion time, coupled with notably long repetition and echo times.                                                  |
+| Inplane T1                                 | inplaneT1        | In arbitrary units (abitrary). T1 weighted structural image matched to a functional (task) image.                                                                                  |
+| Inplane T2                                 | inplaneT2        | In arbitrary units (abitrary). T2 weighted structural image matched to a functional (task) image.                                                                                  |
+| PD and T2 weighted images                  | PDT2             | In arbitrary units (abitrary). PDw and T2w images acquired using a dual echo FSE sequence through view sharing process ([Johnson et al. 1994](https://pubmed.ncbi.nlm.nih.gov/8010268/)).                                      |
```

Comment:
Should this be `PDT2w` to follow other patterns?

Reply:
@tsalo `w` can be omitted for this particular suffix. We kept this as is on demand.

Reply:
Thanks for the clarification. Is the same true for the inplane T1/T2 scans?

Reply:
@tsalo yes, they appear to be used commonly, no changes :)